<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fine_1</class>
 <widget class="QWidget" name="Fine_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Штраф</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Итого к оплате:</string>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buy">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ОПЛАТИТЬ</string>
   </property>
  </widget>
  <widget class="QLabel" name="d">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_start_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
  <widget class="QPushButton" name="history">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>История штрафов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
